### ISSUE TITLE
False positive prnt.sc & www.prnt.sc

### ISSUE THREAD

**Issue by ryanocf** [RPiList/specials#634]
Hey,

`prnt.sc` & `www.prnt.sc` sind gekürzte URL's vom Service Lightshot.
Auch unter der domain prntscr.com zu erreichen.
Die Liste in der es vorkommt: https://raw.githubusercontent.com/RPiList/specials/master/Blocklisten/Phishing-Angriffe

Ich glaube die eigentliche domain die man blocken wollte ist: `pnrtscr.com`
Diese ist ein Scarelink die nur dafür gedacht ist, um Menschen zu erschrecken.
Daher am besten auch gar nicht erst aufrufen.

Zu dieser Seite gibt es wohl auch eine Github repo.
https://github.com/mathhulk/pnrtscr.com

**Comment by ryanocf** [comment]
Lightshot ist ein Tool womit man Screenshots erstellen und hochladen kann.
Diese werden dann über die Domain `prnt.sc` hochgeladen.

Ich habe es jetzt selbst nochmal getestet und wenn man einen screenshot auf `prnt.sc` hochlädt, bekommt man auch eine short URL von der selben Domain zugewiesen zum kopieren.
http://prnt.sc/1qi30h0

Nach Aufruf der oben genannten short URL, wird man auf folgenden link weitergeleitet:
https://image.prntscr.com/image/6eFKN7KHR_unbFp4Ge1fNQ.png

Somit sollte prnt.sc ebenfalls entfernt werden, da es essenziell ist zum Anzeigen hochgeladener Screenshots.

**Comment by ryanocf** [comment]
Hey,

die Domains `prnt.sc` & `www.prnt.sc` sind wieder auf der Liste.
@RPiList

**Comment by RPiList** [comment]
Ja, Phishstats.info meldet die Domain derzeit noch als kompromittiert. Insoweit bleibt die Domain erst mal drauf. Prüfe ich aber in regelmäßigen Abständen, ob die ihre Probleme in den Griff bekommen haben. Viele Grüße.
